fn.go

```diff
@@ -344,7 +410,10 @@ func (f *Function) RunFunction(_ context.Context, req *fnv1beta1.RunFunctionRequ
 				response.Fatal(rsp, errors.Wrapf(err, "error marshaling Unstructured data: %s", err))
```

Comment:
@ca7alindev Don't use fatal in Go microservices, it makes it gone, good that k8s will reload it, but still.